autometrics-macros/src/lib.rs

```diff
@@ -129,6 +130,18 @@ pub fn autometrics(
     output.into()
 }
 
+#[proc_macro_derive(GetLabel, attributes(autometrics))]
```

Comment:
Well, the Serde derives are called `Serialize` and `Deserialize` but the field annotations are still called `serde` because you're configuring how Serde deals with them. I would argue that it's the same here: They're field annotations specific to Autometrics.

Reply:
Good point. The one other difference I see here is that we have another `autometrics` macro that uses the `autometrics` attribute, but the parameters to these two macros are entirely unrelated. Maybe that's still fine?

Reply:
Yes, I would say that we need to document that regardless though. I'm not sure how discoverable these annotations are going to be otherwise (autocomplete seems a bit shaky/non-existent on them).

Reply:
What about calling the whole thing `AutometricsLabel` / `autometrics_label(key = "...")`? It's a bit long but 🤷

Reply:
OK, I'll change it.